Makefile

```diff
@@ -20,6 +20,8 @@ bucket:
 
 test:
 	docker-compose run composer ./vendor/bin/simple-phpunit
+	docker-compose up -d php
+	docker-compose exec php vendor/bin/behat
```

Comment:
```suggestion
	docker-compose run --rm php vendor/bin/behat
```
I think this does just the same in one single command, please tell me if I'm wrong.

Reply:
Just run php is enough in fact